Resources/Prototypes/Roles/Jobs/Security/detective.yml

```diff
@@ -7,12 +7,14 @@
   - !type:DepartmentTimeRequirement
     department: Security
     time: 54000 # 15 hours

```

Comment:
having sec department time for a non-sec role is kinda strange and it should probably be reduced a bit.

Reply:
not necessarily, Detective is still a sec-related role and people need to have played at least some sec to know how to handle situations. For example, you cant just perma/kill someone because they bolted a door. The det needs to know how to handle these various intricate situations, and appropriate sec play-time will allow for this. 15 hours makes sense, since they'd need to have some sec background